clouddriver-openstack/src/main/java/org/openstack4j/api/Builders.java

```diff
@@ -0,0 +1,1166 @@
+import org.openstack4j.openstack.networking.domain.ext.NeutronFirewallRuleUpdate;
+import org.openstack4j.openstack.networking.domain.ext.NeutronFirewallUpdate;
+import org.openstack4j.openstack.networking.domain.ext.NeutronHealthMonitor;
+import org.openstack4j.openstack.networking.domain.ext.NeutronHealthMonitorAssociate;
+import org.openstack4j.openstack.networking.domain.ext.NeutronHealthMonitorUpdate;
+import org.openstack4j.openstack.networking.domain.ext.NeutronHealthMonitorV2;
+import org.openstack4j.openstack.networking.domain.ext.NeutronHealthMonitorV2Update;
+import org.openstack4j.openstack.networking.domain.ext.NeutronLbPool;
+import org.openstack4j.openstack.networking.domain.ext.NeutronLbPoolUpdate;
+import org.openstack4j.openstack.networking.domain.ext.NeutronLbPoolV2;
+import org.openstack4j.openstack.networking.domain.ext.NeutronLbPoolV2Update;
+import org.openstack4j.openstack.networking.domain.ext.NeutronListenerV2;
+import org.openstack4j.openstack.networking.domain.ext.NeutronListenerV2Update;
+import org.openstack4j.openstack.networking.domain.ext.NeutronLoadBalancerV2;
+import org.openstack4j.openstack.networking.domain.ext.NeutronLoadBalancerV2Update;
+import org.openstack4j.openstack.networking.domain.ext.NeutronMember;
+import org.openstack4j.openstack.networking.domain.ext.NeutronMemberUpdate;
+import org.openstack4j.openstack.networking.domain.ext.NeutronMemberV2;
+import org.openstack4j.openstack.networking.domain.ext.NeutronMemberV2Update;
+import org.openstack4j.openstack.networking.domain.ext.NeutronSessionPersistence;
+import org.openstack4j.openstack.networking.domain.ext.NeutronVip;
+import org.openstack4j.openstack.networking.domain.ext.NeutronVipUpdate;
+import org.openstack4j.openstack.sahara.builder.SaharaBuilders;
+import org.openstack4j.openstack.sahara.domain.SaharaCluster;
+import org.openstack4j.openstack.sahara.domain.SaharaClusterTemplate;
+import org.openstack4j.openstack.sahara.domain.SaharaDataSource;
+import org.openstack4j.openstack.sahara.domain.SaharaJob;
+import org.openstack4j.openstack.sahara.domain.SaharaJobBinary;
+import org.openstack4j.openstack.sahara.domain.SaharaJobConfig;
+import org.openstack4j.openstack.sahara.domain.SaharaJobExecution;
+import org.openstack4j.openstack.sahara.domain.SaharaNodeGroup;
+import org.openstack4j.openstack.sahara.domain.SaharaNodeGroupTemplate;
+import org.openstack4j.openstack.sahara.domain.SaharaServiceConfig;
+import org.openstack4j.openstack.storage.block.builder.CinderBuilders;
+import org.openstack4j.openstack.storage.block.domain.CinderBlockQuotaSet;
+import org.openstack4j.openstack.storage.block.domain.CinderVolume;
+import org.openstack4j.openstack.storage.block.domain.CinderVolumeSnapshot;
+import org.openstack4j.openstack.telemetry.builder.CeilometerBuilders;
+import org.openstack4j.openstack.telemetry.domain.CeilometerAlarm;
+
+/**
+ * A utility class to quickly access available Builders within the OpenStack API
+ *
```

Comment:
Can you add a TODO here to describe when this file should be removed?  Assuming you needed to implement some extra features in openstack4j client.